Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
     "cli",
     "codegen",
     "examples",
+    "examples/wasm_examples",
```

Comment:
Please exclude this from the subxt workspace to avoid bring in all these dependencies as these are just for the example